### PR TITLE
Style improvements

### PR DESCRIPTION
**Pull request by linasburneika**
- Added button styling in nav-bar above editor for "show readme" and "transpile" buttons.
- Added shadow hover on preselecting gallery items and cursor type pointer

### CODE REVIEW

File: src/Components/SampleEditor/TabNavigation/TabNavigation.tsx [hunk 1]
```diff
@@ -46,10 +48,12 @@ export class TabNavigation extends Component<TabNavigationProps, TabNavigationSt
     const executable = FeatureToggleClient.isFeatureEnabled(featureFlags.enableEditor);
     return (
       <TabNav showClose={false}>
-        <TabNavigationAction onClick={this.props.onShowReadme}>
-          <div className="icon icon-info" style={this.props.showReadme ? { display: "inline-block", color: "white" } : { display: "inline-block" }}></div>
-        </TabNavigationAction>
-        {executable && <RunCodeButton style={{ paddingLeft: "10px", paddingRight: "10px" }} onRunStarted={this._onRunStarted} onBundleError={this._onBundleError} onRunCompleted={this._onRunCompleted} buildOnRender={false} />}
+        <div className="action-item " onClick={this.props.onShowReadme}>
```
Comment (williamkbentley) [inline src/Components/SampleEditor/TabNavigation/TabNavigation.tsx:51]: Should these changes go in @bentley/monaco-editor @StefanRetief. Or is this okay for now

Comment (StefanRetief) [reply src/Components/SampleEditor/TabNavigation/TabNavigation.tsx:51]: I'll apply the same changes in the editor and it should apply in the next update, for the time being, this should be okay.

File: src/Components/SampleGallery/SampleGallery.scss [hunk 2]
```diff
@@ -47,23 +47,39 @@
...
+.gallery-card-body::after {
+  content: "";
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  box-shadow: inset 0 0 8px rgba(0, 0, 0, 0.75);
+  transition: ease-in-out 0.05s;
```
[the remark this answers is not in the document]
Comment (linasburneika) [reply src/Components/SampleGallery/SampleGallery.scss:74]: 0.2s feels very sluggish for preselect

File: src/Components/SampleGallery/SampleGallery.scss [hunk 3]
```diff
@@ -47,23 +47,39 @@
...
 
+.gallery-card-body::after {
+  content: "";
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  box-shadow: inset 0 0 8px rgba(0, 0, 0, 0.75);
```
[the remark this answers is not in the document]
Comment (linasburneika) [reply src/Components/SampleGallery/SampleGallery.scss]: Changed hover to be pale blue for gallery items

**Review by williamkbentley — APPROVED** [review]
LG

**Pull request merged**